docs/manuals/administrator-manual.md

```diff
@@ -0,0 +1,82 @@
+# Beheerdershandleiding ZGW Office Add-In
+| versie | datum      | wijziging                                                      |
+|--------|------------|----------------------------------------------------------------|
+| 0.1    | 2025-07-07 | Opzet van deze beheerdershandleiding in de context van MS Word |
+
+## Inleiding
+Deze beheerdershandleiding is bedoeld voor systeembeheerders van gemeenten die de ZGW Office Add-In beheren. De 
+add-in is een hulpmiddel dat de integratie van Microsoft Office applicaties met een zaaksysteem gebaseerd op de ZGW 
+APIs mogelijk maakt.
+
+## Uitgangssituatie
+De uitgangssituatie waar deze handleiding op is gebaseerd, is dat de systeembeheerder werkt met Microsoft 365 Office 
+voor een gemeente die de ZGW Office Add-In heeft geïnstalleerd op een omgeving die toegankelijk is voor de medewerkers 
+van de gemeente.
+
+Om deze uitgangssituatie te bereiken, is het noodzakelijk dat de ZGW Office Add-In componenten geïnstalleerd en 
+geconfigureerd zijn op het Applications Platform van de gemeente. Zie voor meer informatie de 
+[Installatie van de ZGW Office Add-In componenten](./platform-installation-manual.md) documentatie.
+
+### Overzicht van de ZGW Office Add-In componenten
+De ZGW Office Add-In bestaat uit verschillende componenten die samenwerken om de integratie tussen Microsoft Word en de
+ZGW APIs mogelijk te maken. Hieronder [C4 Context Diagram](https://c4model.com/diagrams/system-context) met de 
+belangrijkste componenten:
+```mermaid
+C4Context
+    Enterprise_Boundary(o, "Employee Workspace") {
+        System_Boundary(word, "Microsoft Word") {
+            System(WordApp, "Microsoft Word Application")
+            System(WordWebApp, "Microsoft Word Web Application")
+            System(WordAddIn, "ZGW Office Add-In")
+        }
+    }
+
+    Enterprise_Boundary(b0, "Applications Platform") {
+        System(OfficeProxy, "ZGW Office Frontend Proxy")
+        System(OfficeService, "ZGW Office Backend Service")
+        System(Objecten, "Objecten API")
+        System(OpenZaak, "ZGW API")
+    }
+
+    Rel(WordApp, WordAddIn, "<include>")
+    Rel(WordWebApp, WordAddIn, "<include>")
+    Rel(WordAddIn, OfficeProxy, "<use>")
+    Rel(OfficeProxy, OfficeService, "<redirect>")
+    Rel(OfficeService, Objecten, "<use>")
+    Rel(OfficeService, OpenZaak, "<use>")
+
+    UpdateElementStyle(WordAddIn, $bgColor="red", $borderColor="red")
+    UpdateElementStyle(OfficeProxy, $bgColor="red", $borderColor="red")
+    UpdateElementStyle(OfficeService, $bgColor="red", $borderColor="red")
+
+    UpdateLayoutConfig($c4ShapeInRow="3", $c4BoundaryInRow="1")
+```
+
+#### Componenten van de ZGW Office Add-In
+De componenten van de ZGW Office Add-In zijn rood gemarkeerd in het diagram. Hieronder een overzicht van wat deze 
+componenten doen en hoe ze samenwerken.
+
+##### ZGW Office Add-In
+De ZGW Office Add-In is de add-in die geïnstalleerd is in de Microsoft Word desktop en webversie. Deze add-in is 
```

Comment:
So this means as an admin you need to install it twice? Once for desktop and once for web? That's not true is it? I thought you only needed to install it once? I think you mean here that you only need to install it once but that it then becomes available for both the desktop and web version? Maybe rephrase this a bit?